docs/components/WizardLayoutView.jsx

```diff
@@ -144,6 +144,7 @@ export default class WizardLayoutView extends React.PureComponent {
               stepper={stepper}
               subtitle="Ensure a smooth upcoming school year by following a few easy steps below."
               title="Back to school guide"
+              fullscreen={false}
```

Comment:
isn't this the same as just not including it?

Reply:
true facts